lib/sorbet-rails/model_plugins/active_record_finder_methods.rb

```diff
@@ -60,7 +63,10 @@ def create_finder_methods_for(class_rbi, class_method:, include_methods_that_ret
         "find_or_create_by",
         parameters: [
           Parameter.new("attributes", type: "T.untyped") ,
-          Parameter.new("&block", type: "T.proc.params(object: #{self.model_class_name}).void"),
+          Parameter.new(
+            "&block",
+            type: "T.nilable(T.proc.params(object: #{self.model_class_name})).void"
```

Comment:
I think you typo'd on some of these, they're not generating valid RBIs:
```suggestion
            type: "T.nilable(T.proc.params(object: #{self.model_class_name}).void)"
```